ext/drcovlib/drcovlib.c

```diff
@@ -213,7 +213,7 @@ static void
 dump_drcov_data(void *drcontext, per_thread_t *data)
 {
     version_print(data->log);
-    module_table_print(module_table, data->log, false);
+    module_table_print(module_table, data->log, options.verbose_module_table);
```

Comment:
@zhaoqin Right, I hadn't considered that. May I assume that nothing relies on the current format of verbose module table dump and modify it so that the post-processing still works?